eXo/Sources/Controllers/HomePage/HomePageViewController.swift

```diff
@@ -249,10 +238,26 @@ class HomePageViewController: eXoWebBaseController, WKNavigationDelegate, WKUIDe
         if !UIApplication.shared.isNetworkActivityIndicatorVisible {
             UIApplication.shared.isNetworkActivityIndicatorVisible = true;
         }
+        // Display the navigation bar at login or other pages accessible for anonymous users && display the bar when luser is logged in
+        // Home Page Address: portal/dw
+        if let urlRequest = request.url {
+            if let urlComponent = URLComponents(string: urlRequest.absoluteString) {
+                if (urlComponent.path == "/portal/dw/"){
```

Comment:
URL couldn't be different than /portal/dw/ ? /portal/dw for example, or /portal/g/:spaces:support/ for example